tough/src/editor/signed.rs

```diff
@@ -591,26 +659,46 @@ trait TargetsWalker {
     /// Walks a given directory and calls the provided function with every file found.
     /// The function is given the file path, the output directory where the user expects
     /// it to go, and optionally a desired filename.
-    fn walk_targets<F>(
+    async fn walk_targets<F>(
         &self,
         indir: &Path,
         outdir: &Path,
-        f: F,
+        mut f: F,
```

Comment:
Why is `f` `mut` now?

Reply:
needed to invoke a `FnMut`, which I chose because it's the least restricting